qlty-check/src/executor/invocation_result.rs

```diff
@@ -296,9 +305,10 @@ impl InvocationResult {
                 .with_context(|| format!("Failed to read staged file {:?}", &staged_path))?;
 
             if workspace_contents != staged_contents {
+                info!("Rewriting file {:?}", &workspace_path);
```

Comment:
Nice. This is a highly valuable log line to have and I think info is the right level given the importance of the operation (and relative small number)